app/helpers/change_description_helper.rb

```diff
@@ -0,0 +1,88 @@
+module ChangeDescriptionHelper
```

Comment:
Change description doesn't make a lot of sense when you read it.. This file contains functions used for versions. What about we call it versions_helper instead?